scrapper.py

```diff
@@ -75,7 +110,16 @@ def parse(self):
         """
         Parses each article
         """
-        pass
+        headers = {
```

Comment:
it is not correct. This method should be called from _fill_articles_with_meta_information method, otherwise you make excessive requests